static/css/frame.scss

```diff
@@ -47,6 +47,61 @@
   }
 }
 
+.promo-panel {
+  .edit {
+    right: 50px;
+    top: 75px;
+  }
+  .promo-box {
+    border-radius: 3px;
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.3), 0 0 20px rgba(0, 0, 0, 0.2);
+    position: absolute;
+    width: 180px;
+    z-index: 999;
+    padding: 10px 5px;
+    text-align:center;
+  }
+
+  .title {
+    font-size: 14px;
+    line-height: 14px;
+    font-weight: bold;
+  }
+
+  .message {
+    display: inline-block;
```

Comment:
I don't think you need `inline-block` here. (Was there a particular problem you were trying to solve?)

Reply:
I think it was to support the longer message string and  previous UI of hand wave emoji, tried long string with title as text and it works